deploy/ocp-templates/heketi-template.yaml

```diff
@@ -121,3 +123,6 @@ parameters:
   displayName: heketi fstab path
   description: Set the fstab path, file that is populated with bricks that heketi creates
   value: /var/lib/heketi/fstab
+- name: HEKETI_IGNORE_STALE_OPERATIONS
+  displayName: Heketi ignore stale operations
```

Comment:
"Heketi ignore stale operations" --> "Whether to ignore stale operations at startup" ?